components/eam/src/chemistry/mozart/chemistry.F90

```diff
@@ -1643,19 +1644,20 @@ subroutine chem_timestep_tend( state, ptend, cam_in, cam_out, dt, pbuf,  fh2o, f
 !-----------------------------------------------------------------------
     do k = 1,pver
        cldw(:ncol,k) = state%q(:ncol,k,ixcldliq) + state%q(:ncol,k,ixcldice)
+       cldw_liq(:ncol,k) = state%q(:ncol,k,ixcldliq)
        if (ixndrop>0) &
             ncldwtr(:ncol,k) = state%q(:ncol,k,ixndrop)
     end do
 
     call t_startf( 'chemdr' )
-    call gas_phase_chemdr(lchnk, ncol, imozart, state%q, &
-                          state%phis, state%zm, state%zi, calday, &
-                          state%t, state%pmid, state%pdel, state%pdeldry, state%pint, &
-                          cldw, tropLev, ncldwtr, state%u, state%v, &
-                          chem_dt, state%ps, xactive_prates, &
-                          do_cloudj_photolysis, do_cloudj_clouds, do_cloudj_aerosols, &
-                          do_cloudj_lookup_diag, do_cloudj_aerosol_diag, do_cloudj_nocloud_diag, &
-                          fsds, cam_in%ts, cam_in%asdir, cam_in%ocnfrac, cam_in%icefrac, &
+    call gas_phase_chemdr(lchnk, ncol, imozart, state%q,                                              &
+                          state%phis, state%zm, state%zi, calday,                                     &
+                          state%t, state%pmid, state%pdel, state%pdeldry, state%pint,                 &
+                          cldw, cldw_liq, tropLev, ncldwtr, state%u, state%v,                         &
```

Comment:
Is the total cloud water (cldw) used in the fixed code? My understanding was that cldw needs to be replaced by cldw_liq. If not necessary, cldw should not be passed to the module.

Reply:
The total cloud water cldw should still be passed to the module since it is used by the photolysis rate calculation.